src/views/RandomView.vue

```diff
@@ -25,8 +25,13 @@ import { ref } from "vue"
       const ingredientsList = [];
       for (let i = 1; i < 16; i++) {
         if (data.drinks[0][`strIngredient${i}`]) {
-          const ingredient = data.drinks[0][`strMeasure${i}`] + " " + data.drinks[0][`strIngredient${i}`]
-          ingredientsList.push(ingredient)
+          if (!data.drinks[0][`strMeasure${i}`]) {
```

Comment:
This is such a rad way to check that it exists and make sure the right measurement goes with the right ingredient.